### PR TITLE
Update changelog for 2.0.0

### PR DESCRIPTION
**Pull request by jsteiner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,41 @@ complete changelog, see the git history for each version via the version links.
...
+- Fix an issue where setting an association to `nil` would break `insert` ([#193])
+- Fix an issue where unbuild has_many through associations were not removed in
+  `params_*` functions ([#192])
+
+[2.0.0]: https://github.com/thoughtbot/ex_machina/compare/v1.0.2...v2.0.0
+[#149]: https://github.com/thoughtbot/ex_machina/pull/149
+[#151]: https://github.com/thoughtbot/ex_machina/pull/151
+[#148]: https://github.com/thoughtbot/ex_machina/pull/148
+[#147]: https://github.com/thoughtbot/ex_machina/pull/147
+[#151]: https://github.com/thoughtbot/ex_machina/pull/168
```
Comment (paulcsmith) [inline CHANGELOG.md]: This should be `168`

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,41 @@ complete changelog, see the git history for each version via the version links.
...
+[#174]: https://github.com/thoughtbot/ex_machina/pull/174
```
Comment (paulcsmith) [inline CHANGELOG.md]: It looks like this isn't used

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,41 @@ complete changelog, see the git history for each version via the version links.
...
+  schema, you can set the value to `0` in your factory and it will automatically
+  cast the value to a Decimal.
+
+- Add `string_params_for`, which is useful for controller specs. ([#168])
+- Add `Sequence.reset/0` for reseting sequences between tests. ([#151])
+
+### Changed
+
+- `params_*` functions now drop fields with `nil` values ([#148])
+- Don't delete `has_many`s from `params_*` functions ([#148])
```
Comment (paulcsmith) [inline CHANGELOG.md]: Should these link to different PRs?

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,41 @@ complete changelog, see the git history for each version via the version links.
...
+### Added
+
+- Cast all values before insert ([#149])
+
+  For example, this means that if you have `field :equity, :decimal` in your
+  schema, you can set the value to `0` in your factory and it will automatically
+  cast the value to a Decimal.
+
+- Add `string_params_for`, which is useful for controller specs. ([#168])
+- Add `Sequence.reset/0` for reseting sequences between tests. ([#151])
```
Comment (paulcsmith) [inline CHANGELOG.md]: resetting should have two `t`s

Comment (paulcsmith) [inline CHANGELOG.md]: Also I believe this links to the wrong PR

**Comment by paulcsmith** [comment]
Ping @jsteiner in case you forgot about this. I can tackle it if you don't have time, just LMK

**Comment by jsteiner** [comment]
@paulcsmith I was going to hold off until we're fully ready for 2.0. Let's discuss tomorrow.

**Pull request merged**